pxd.c

```diff
@@ -1024,13 +1024,14 @@ static void pxd_rq_fn(struct request_queue *q)
 		fproot = &req->fproot;
 		fp_root_context_init(fproot);
 
+#ifdef __PX_FASTPATH__
 		if (pxd_dev->fp.fastpath) {
 			// route through fastpath
-			INIT_WORK(&fproot->work, fp_handle_io);
```

Comment:
since default action on work queue is processing IO, init work is moved to fp_root_context_init() method.